packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -485,6 +480,8 @@ export class LoggingError extends Error {
         for (const key of Object.getOwnPropertyNames(this)) {
             props[key] = this[key];
         }
+        // Remove pii member
+        delete (props as unknown as LoggingError).pii;
```

Comment:
delete is expensive in JS, assign undefined value instead

Reply:
Ok interesting.  If I assign `undefined`, there's a compiler error (warning as error?) that `pii` is never read :(